scripts/Version.js

```diff
@@ -17,16 +18,17 @@ function Version(v) {
 
 Version.fromBranch = function (branch) {
   var m;
-
-  // master === the highest version number
-  if (branch === 'master') return new Version('5.0.0');
+  var majorMinorRegex = /^\d+\.\d+$/;
 
   // n.m -> n.m.0
-  if (m = branch.match(/^\d+\.\d+$/)) return new Version(branch + '.0');
+  if (m = branch.match(majorMinorRegex)) return new Version(branch + '.0');
 
   // n.x -> n.(maxVersion + 1).0
   if (m = branch.match(/^(\d+)\.x$/i)) return maxMinorVersion(m[1]).increment('minor');
 
+  var branchVersion = branchVersions[branch];
```

Comment:
We might want to use this before the other cases.. on second thought, lemme change this...